flow/hash.go

```diff
@@ -38,13 +36,13 @@ func (fl *ICMPLayer) Hash(hasher hash.Hash) {
 }
 
 // Hash computes the hash of a transport layer
-func (tl *TransportLayer) Hash(hasher hash.Hash) {
+func (tl *TransportLayer) Hash(hasher hash.Hash, way bool) {
 	if tl == nil {
 		return
 	}
 
 	value32 := make([]byte, 4)
-	if tl.A > tl.B {
+	if way {
```

Comment:
Does Skydive handle traffic between an IP to itself? 

Consider a TCP connection from `127.0.0.1:1000` to `127.0.0.1:2000`.  Since the network endpoints are identical, we'll have `way = false` for both directions. I think it means that packets from port 1000 to 2000 will get a different transport layer hash from packets going from 2000 to 1000.